templates/cli/lib/utils.js.twig

```diff
@@ -14,6 +18,56 @@ function getAllFiles(folder) {
     return files;
 }
 
+function showConsoleLink(serviceName, action, open, id = '') {
+    let resource = '';
+    let service = '';
+
+    switch (serviceName) {
+        case "account":
+            service = 'account';
+            break;
+        case "databases":
+            resource = 'database';
+            service = 'databases';
+            break;
+        case "functions":
+            resource = 'function';
+            service = 'functions';
+            break;
+        case "projects":
+            service = `project-${id}`;
+            id = '';
+            break;
+        case "teams":
+            resource = 'team';
+            service = 'auth/teams';
+            break;
+
+        case "users":
+            resource = 'user';
+            service = 'auth';
+            break;
+        default:
+            return;
+    }
+
+    const baseUrl = globalConfig.getEndpoint().replace('/v1', '');
+
+    const end = action === 'get' ? (id ? `/${resource}-${id}` : `/${resource}`) : '';
+    const projectId = localConfig.getProject().projectId;
+    const middle = resource !== '' ? `/project-${projectId}` : '';
+    const url = `${baseUrl}/console${middle}/${service}${end}`
+
+
+    success(url);
+
+    if (open) {
+        const start = (process.platform == 'darwin' ? 'open' : process.platform == 'win32' ? 'start' : 'xdg-open');
```

Comment:
Do we have a fallback for if this fails? Maybe on a entirely CLI environment like a server or CI we could display a "If your browser does not automatically open please copy the link 'xxxxxxxxx'"